docs/recording/trace_filtering.md

```diff
@@ -34,25 +34,27 @@ It will be handy to prepare a trace filter setting file like the following.
 
 export CARET_IGNORE_NODES=\
 "\
-/rviz*\
+/rviz*:\
+/care_trace_*:\
```

Comment:
It may be typo (caret_trace_*)

Reply:
Thank you for pointing it out :sweat_smile: 
I pushed a new commit to fix it.